api/src/main/java/com/pip/chatbot/exception/CustomExceptionHandler.java

```diff
@@ -0,0 +1,20 @@
+package com.pip.chatbot.exception;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.client.HttpClientErrorException;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import java.util.HashMap;
+
+@ControllerAdvice
+public class CustomExceptionHandler extends ResponseEntityExceptionHandler {
+    @ExceptionHandler(HttpClientErrorException.BadRequest.class)
+    public ResponseEntity<?> handleBadRequestException(HttpClientErrorException.BadRequest e, WebRequest webRequest) {
+        return ResponseEntity
+                .status(400)
+                .body(new HashMap.SimpleEntry<>("message", "Bad request"));
```

Comment:
1.Łapiemy wszystkie wyjątki
2. Zrobić klaskę ErrorResponse z polem List<String> errors i zwracać ResonseEntity\<ErrorResponse\> 
3..Jeżeli wyjątek to ChatbotException to rzutować na niego pobierać i i List\<String\> errors i wpisywać do ErrorResponse.Zwracać kod 400
4. W każdym innym przypadku zwracamy 500 i message z Exception'a.